components/camel-workday/src/test/java/org/apache/camel/WorkdayComponentConfigurationTest.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.camel;
+
+import org.apache.camel.component.workday.WorkdayComponent;
+import org.apache.camel.component.workday.WorkdayConfiguration;
+import org.apache.camel.component.workday.WorkdayEndpoint;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Test;
+
+public class WorkdayComponentConfigurationTest extends CamelTestSupport {
+
+    @Test
+    public void createProducerMinimalConfiguration() throws Exception {
+
+        WorkdayComponent workdayComponent = context.getComponent("workday", WorkdayComponent.class);
+
+        WorkdayEndpoint workdayEndpoint = (WorkdayEndpoint)workdayComponent.createEndpoint("workday-raas:/<Owner>/<ReportName>?" +
+        "host=impl.workday.com" +
+                "&tenant=camel" +
+                "&clientId=f7014d38-99d2-4969-b740-b5b62db6b46a" +
```

Comment:
Could this stuff be public and published in this way?

Reply:
No real data, Used only to validate the configuration in the test.